collaboration-service/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM node:17-alpine
```

Comment:
Our project uses node 18 LTS, not sure if downgrading will cause any compatibility issues

Reply:
Okay, use node 18